app/assets/javascripts/app/views/single-post-viewer/single_post_moderation.js

```diff
@@ -0,0 +1,47 @@
+app.views.SinglePostModeration = app.views.Feedback.extend({
```

Comment:
This view is only the little part of the screenshots where the 3 icons [hide, block, report] (or 1 delete icon) is displayed. So: There is already a screenshot. :)